skt/reporter.py

```diff
@@ -283,22 +283,29 @@ def getmergeinfo(self):
         Returns: A list of strings representing data about applied patches and
                  base repository.
         """
-        result = ['We applied the following patch']
-        if len(self.mergedata['localpatch'] + self.mergedata['patchwork']) > 1:
-            result[0] += 'es:\n'
-        else:
-            result[0] += ':\n'
+        patchlist = self.mergedata['localpatch'] + self.mergedata['patchwork']
 
-        for patchpath in self.mergedata['localpatch']:
-            result += ['  - %s' % patchpath]
+        if patchlist:
+            result = ['We applied the following patch']
+            if len(patchlist) > 1:
+                result[0] += 'es:\n'
+            else:
+                result[0] += ':\n'
 
-        for (purl, pname) in self.mergedata['patchwork']:
-            result += ['  - %s,' % pname,
-                       '    grabbed from %s\n' % purl]
+            for patchpath in self.mergedata['localpatch']:
+                result += ['  - %s' % patchpath]
 
-        result += ['on top of commit %s from the repository at' %
-                   self.mergedata['base'][1][:12],
-                   '  %s' % self.mergedata['base'][0]]
+            for (purl, pname) in self.mergedata['patchwork']:
+                result += ['  - %s,' % pname,
+                           '    grabbed from %s\n' % purl]
+
+            result += ['on top of commit %s from the repository at' %
+                       self.mergedata['base'][1][:12],
+                       '  %s' % self.mergedata['base'][0]]
+        else:
+            result = ['We grabbed git tree with HEAD at %s from the '
```

Comment:
Perhaps: `We cloned the git tree and checked out %s from the repository at...`. Using `cloned` allows us to be a bit more specific than `grabbed`. ;)